src/vaults/harvests.utils.ts

```diff
@@ -205,6 +211,7 @@ async function loadEventYieldData(
 ): Promise<VaultHarvestData[]> {
   const { vaults } = await chain.getSdk();
   const { address, version } = vault;
+  console.log(`[${vault.name}]: Load Event Yield Data`);
```

Comment:
it looks like this is desired but wanted to double check

Reply:
this was for debugging, we can remove it if desired. shall we?